build/bazelutil/check.sh

```diff
@@ -56,7 +56,6 @@ pkg/cmd/cockroach-oss/BUILD.bazel
 pkg/cmd/bazci/githubpost/BUILD.bazel
```

Comment:
Oh, it looks like a lot of these broken tests have actually been fixed by now. Can you additionally remove every file except `pkg/cmd/prereqs/BUILD.bazel` from this list? It looks like all the rest are taken care of. :)

This is great since `pkg/cmd/prereqs` has no use in the Bazel world, so we can just delete the package instead of fixing the test.